### ISSUE TITLE
End user experience issue with GUI prompts rather than on command line

### ISSUE THREAD

**Issue by TheFitITGuy** [adamdriscoll/poshtools#554]
This seems to be an issue for me as I'm trying to write scripts that will be used by running the ps1 file with PowerShell. When developing I want to use Visual Studio but not all machines will have it installed that I will run it on. It shows in a GUI window when running the scripts. As shown below: 
![image](https://user-images.githubusercontent.com/25332879/29742414-d436b250-8a33-11e7-9ebb-7aae30eb3f2a.png)

When I run in PowerShell ISE or PowerShell I see it on the console line. As shown below: 
![image](https://user-images.githubusercontent.com/25332879/29742421-f4757452-8a33-11e7-8d6b-8bb4f2a4be7c.png)

I want to be able to see what an end user would get from running the script if they did not have visual studio installed. This means I am unable to get a true end user experience. I would like the script prompts to show in the PowerShell Interactive Window. Can you please point me to the documentation to change this, add this feature, or provide documentation to change this?

**Comment by henrik-jensen** [comment]
I agree with @TheFitITGuy .
I imaging that this enhancement also, by OP's design requirements, should/would allow running of  System::Console scripts in PowerShell Interactive Windows?

Here is an example that fails (both in PowerShell Interactive Windows and ISE's console):
```powershell
Write-Host "Hello world"
write-host "Press any key to continue..."
[void][System.Console]::ReadKey($true)
```
<img src="https://user-images.githubusercontent.com/9413242/30521691-1e6e8f18-9bc4-11e7-8a11-28f086291457.png" width="75%">
Googlated the red danish to english:

> Exception calling "ReadKey" with "1" argument(s): "It is not possible to read keys when none of the programs have a console or when console inputs have been redirected from a file. Try Console.Read."

PS. I would have suggested you to add the same tags as c# for distinguishing between Console and Win apps, but then I noticed you already do, `<OutputType>Exe</OutputType>` is already sat in .pssproj. In c# `.csproj` files, I think this indicates a Console App with an attached console as opposed to  `<OutputType>WinExe</OutputType>` for Winforms/WpfApps.

**Comment by johnwc** [comment]
Yes, I too need this ability. We need to be able to run a console based script and debug it, seeing what it looks like in a real console and not the VS interactive window.
